kubetest/main.go

```diff
@@ -480,6 +480,10 @@ func prepareGcp(o *options) error {
 			return fmt.Errorf("fail to set project %s : err %v", p, err)
 		}
 
+		if err := os.Setenv("PROJECT", p); err != nil {
```

Comment:
Please put a TODO to stop setting this globally. Instead we should start using pushEnv() locally where it is consumed.